src/Compilers/CSharp/Portable/Binder/ContextualAttributeBinder.cs

```diff
@@ -15,50 +15,13 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal sealed class ContextualAttributeBinder : Binder
     {
         private readonly Symbol _attributeTarget;
-        private readonly Symbol _attributedMember;
 
         /// <param name="enclosing">Next binder in the chain (enclosing).</param>
         /// <param name="symbol">Symbol to which the attribute was applied (e.g. a parameter).</param>
         public ContextualAttributeBinder(Binder enclosing, Symbol symbol)
             : base(enclosing, enclosing.Flags | BinderFlags.InContextualAttributeBinder)
         {
             _attributeTarget = symbol;
-            _attributedMember = GetAttributedMember(symbol);
-        }
-
-        /// <summary>
-        /// We're binding an attribute and this is the member to/in which the attribute was applied.
-        /// </summary>
-        /// <remarks>
-        /// Method, property, event, or null.
-        /// A virtual property on Binder (i.e. our usual pattern) would be more robust, but the applicability
-        /// of this property is so narrow that it doesn't seem worthwhile.
-        /// </remarks>
-        internal Symbol AttributedMember
```

Comment:
@jcouv your nameof-parameter work isn't introducing new dependency on this property, right? You only use the `AttributeTarget`?